### ISSUE TITLE
Incorrect usage info

### ISSUE THREAD

**Issue by IndianBartonka** [aternosorg/mclogs-java#9]
```console
java.lang.NullPointerException: Cannot invoke "gs.mclo.api.MclogsClient.getInstance()" because "this.client" is null
	at gs.mclo.api.response.UploadLogResponse.getUrl(UploadLogResponse.java:34)
	at me.indian.bds.mclog.McLog.sendLog(McLog.java:46)
	at me.indian.bds.BDSAutoEnable.init(BDSAutoEnable.java:125)
	at me.indian.bds.BDSAutoEnable.<init>(BDSAutoEnable.java:97)
	at me.indian.bds.BDSAutoEnable.main(BDSAutoEnable.java:101)
```
	
```java
public class McLog {

    private final BDSAutoEnable bdsAutoEnable;
    private final Logger logger;
    private final Gson gson;
    private final MclogsClient mclogsClient;

    public McLog(final BDSAutoEnable bdsAutoEnable) {
        this.bdsAutoEnable = bdsAutoEnable;
        this.logger = bdsAutoEnable.getLogger();
        this.gson = GsonUtil.getGson();
        this.mclogsClient = new MclogsClient("BDS-Auto-Enable", bdsAutoEnable.getProjectVersion(), bdsAutoEnable.getVersionManager().getLoadedVersion());

        System.out.println(mclogsClient);
        System.out.println(mclogsClient.getInstance());
    }

    public void sendLog() {
        try {
            final Log log = new Log(this.logger.getLogFile().toPath());
            final CompletableFuture<UploadLogResponse> future = this.mclogsClient.uploadLog(log);
            final UploadLogResponse response = future.get();
            System.out.println(response.getUrl());
        } catch (final Exception exception) {
            exception.printStackTrace();
        }
    }
}
```

**Comment by IndianBartonka** [comment]
Version 4.01 dont work but 3.0.1 working good
```java

public class McLog {

    private final BDSAutoEnable bdsAutoEnable;
    private final Logger logger;
    private final MclogsClient mclogsClient;

    public McLog(final BDSAutoEnable bdsAutoEnable) {
        this.bdsAutoEnable = bdsAutoEnable;
        this.logger = bdsAutoEnable.getLogger();
        this.mclogsClient = new MclogsClient("BDS-Auto-Enable", bdsAutoEnable.getProjectVersion(), bdsAutoEnable.getVersionManager().getLoadedVersion());
    }

    public void sendLog() {
        try {
            final Log log = new Log(this.logger.getLogFile().toPath());
            
            final UploadLogResponse response = this.mclogsClient.uploadLog(log);
            
            System.out.println(response.getUrl());
        } catch (final Exception exception) {
            exception.printStackTrace();
        }
    }
}

```



Pls fix that

**Comment by JulianVennen** [comment]
Fixed in 4.0.2
